src/tools/execution-results.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// Shared execution result checking code
+//
+
+namespace wasm {
+
+static void verifyBitwiseEqual(Literal a, Literal b) {
```

Comment:
Maybe `assertBitwiseEqual`? It's harder to see that this can assert/abort now that it has more distance between it and the callsite.

Alternatively: `static bool isBitwiseEqual`, that we call with `assert(isBitwiseEqual(a, b))`, though that makes it harder to weave in the `cout` for non-equal literals.